docs/fr/users/07_Frequently_Asked_Questions.md

```diff
@@ -24,7 +24,7 @@ Le flux peut avoir une syntaxe invalide, il peut ne pas être reconnu par la bib
 Il faut dans un premier temps déterminer la cause du problème.
 Voici la liste des étapes à suivre pour la déterminer :
 
-1. __Vérifier la validité du flux__ grâce à l'[outil en ligne du W3C](http://validator.w3.org/feed/ "Validateur en ligne de flux RSS et Atom"). Si ça ne fonctionne pas, nous ne pouvons rien faire.
+1. __Vérifier la validité du flux__ grâce à l'[outil en ligne du W3C](https://validator.w3.org/feed/ "Validateur en ligne de flux RSS et Atom"). Si ça ne fonctionne pas, nous ne pouvons rien faire.
```

Comment:
You don't have to bother changing the French ones. I'll take care of that as part of https://github.com/FreshRSS/FreshRSS/pull/2590

Reply:
ok.
just trying to find the http links and moving them to https.
sorry if I stepped out.
and if you are responsible for the french one please use https links everywhere you can so that we have a better web.

Reply:
It was just a little note to save you some effort. I would assume there are more in the French.

With #2590 English is the "source". Once I merge your English changes into it, Lokalize (or any translation app) will automatically show that the string was updated while highlighting the changes. So it's just that the French Markdown itself will be completely obsolete soon since it's auto-generated now. ;-)

Nothing procedural about it. :-)

Reply:
ty.
btw should I close this pull request or is this different from bug report?

Reply:
Most of the time you should just keep adding to a PR.